### PR TITLE
feat: 🎸 Added list view for workers

### PR DESCRIPTION
**Pull request by ZedLi**
✅ Closes: ICU-5251

:tickets: [Jira ticket](https://hashicorp.atlassian.net/browse/ICU-5251)

## Description
List view for workers

### Screenshots (if appropriate):

image

### CODE REVIEW

File: ui/admin/tests/acceptance/workers/list-test.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+  test('Users can navigate to workers with proper authorization', async function (assert) {
+    assert.expect(2);
+    await visit(urls.globalScope);
+    assert.ok(
+      instances.scopes.global.authorized_collection_actions.workers.includes(
+        'list'
+      )
+    );
+    assert.dom(`[href="${urls.workers}"]`).isVisible();
```
[the remark this answers is not in the document]
Comment (cameronperera) [reply ui/admin/tests/acceptance/workers/list-test.js:45]: I like this approach. It matches well with our approach to making our acceptance tests as close to what the user sees and how they interact with the app.

**Review by cameronperera — APPROVED** [review]
LGTM!

**Review by lisbet-alvarez — APPROVED** [review]

**Review by dvonanderson — APPROVED** [review]
LGTM!

File: addons/api/addon/models/worker.js [hunk 2]
```diff
@@ -4,6 +4,23 @@ import { fragment } from 'ember-data-model-fragments/attributes';
 export default class WorkerModel extends GeneratedWorkerModel {
   // =attributes
 
+  /**
+   * Names are optional on models in our API.  Thus we need to fallback on ID
+   * for display purposes.
+   * @type {string}
+   */
+  get displayName() {
```
[the remark this answers is not in the document]
Comment (ZedLi) [reply addons/api/addon/models/worker.js:12]: Any thoughts on this @randallmorey?

**Pull request merged**